nose/case.py

```diff
@@ -357,7 +359,7 @@ def __str__(self):
         name = "%s.%s.%s" % (self.cls.__module__,
                              self.cls.__name__,
                              name)
-        if arg:
+        if not arg == '()':
```

Comment:
I thought I posted this before, but apparently not.  The local variable names need to change so that there's a better chance of avoiding this confusion the next time someone reads this code.